apps/calendar/style/multi_day.css

```diff
@@ -393,14 +389,14 @@ html[dir="rtl"] .md__event.is-partial-small .md__event-title + .md__event-locati
   content: '';
   display: block;
   position: absolute;
-  top: 0;
+  top: 0.25rem;
   width: 0;
   height: 0;
   border-style: solid;
-  border-width: 0.4rem 0 0.4rem 0.6rem;
-  border-color: transparent transparent transparent #008EAB;
-  /* scale is for smooth edges and translate for centering the line */
-  transform: scale(0.99) translateY(0.25rem);
```

Comment:
why remove this transform? It may or may not be strictly necessary but seems orthogonal to this BiDi refactor?